### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by Darshan-ko** [SforAiDl/genrl#189]
When I tried running the SAC code on CartPole-v0 (from README.md ), it raises an "AttributeError: 'Discrete' object has no attribute 'high'."
It is due to not allowing discrete action spaces in SAC. 
Also, instead of gym.make(...) for SAC, :
`
from genrl.environments import VectorEnv

env = VectorEnv(...)
`

**Comment by sampreet-arthi** [comment]
The usage shown in the README is old (before a lot of stuff was added). Try:
```
from genrl.environments import VectorEnv
env = VectorEnv("Pendulum-v0")
```
instead.

**Comment by Darshan-ko** [comment]
Ok, thanks.
Can I take up the issue?

**Comment by ajaysub110** [comment]
Sure, go for it! Submit a PR when you're done.

**Comment by Sharad24** [comment]
Resolved in #191
